modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2107,7 +2107,11 @@ export class Wallet implements IWallet {
     if (!this.baseCoin.supportsStaking()) {
       throw new Error(`Staking not supported for ${this.coin()}`);
     }
-    return new StakingWallet(this);
+    const isEthTss =
+      this.baseCoin.getFamily() == 'eth' && this._wallet.coinSpecific?.walletVersion
```

Comment:
Does this need a to check for `teth` and `geth` or are they all returning `eth`?

Reply:
as far as I can tell the coin family is eth